lib/buildkite/builder/github.rb

```diff
@@ -35,6 +39,12 @@ def pull_request_files
         files
       end
 
+      def pull_request
```

Comment:
`Buildkite::Env` also has a `#pull_request` method: https://github.com/Gusto/buildkite-builder/blob/63dd02afddc6a5fbddc4670e49051014e46ddad4/lib/buildkite/env.rb#L39

Different classes so will not conflict, but may be confusing?